src/aviary/message.py

```diff
@@ -124,16 +123,24 @@ def create_message(
         cls,
         role: str = DEFAULT_ROLE,
         text: str | None = None,
-        image: np.ndarray | None = None,
+        images: list[np.ndarray | str] | str | np.ndarray | None = None,
```

Comment:
Can you document what `str`'s meaning is in a docstring? Just mention its a base 64 encoded image